### DISCUSSION TITLE
How to properly set host-id and tags

### DISCUSSION THREAD

**Discussion by thelittlerocket** [smallstep/certificates#524] [Q&A] [answered]
Hi there,
I try to sign a host key via the following command but when I using `step ssh hosts`, the hostname shows up correctly but there is no info in the ID and TAGS columns, would like to know how I can properly set those two fields?
`step ssh certificate "hostname" 'public.key' --sign --host --host-id 'uuid' --provisioner 'provisioner-name' --principal 'hostname' --token 'token' --set 'stage=dev' --set 'type=web'`

I was expecting the following result as the `step ssh hosts` output
HOSTNAME       ID       TAGS
hostname           uuid    stage=dev,type=web

Thanks in advance.

**Comment by tashian** [comment, marked as answer]
Hi @thelittlerocket, unfortunately we don't support host IDs or tags in the open source version of `step`. Our [commercial SSH product](https://smallstep.com/sso-ssh/) has tagging features and it's free when you use it in "single player mode."
